components/VersionSection.tsx

```diff
@@ -56,61 +104,105 @@ export default function VersionSection({
+            You are up to date!
+          </Paragraph>
+        </>
+      )}
+    </div>
+  );
+
   return (
     <SettingsSection
       heading="App Version"
-      controlArea={
-        <div className="flex max-w-52 flex-1 flex-col items-center justify-center text-center">
-          {isLoading && (
-            <div className="flex flex-col items-center space-x-2">
-              <Loader2 className="h-8 w-8 animate-spin" />
-              <Paragraph variant="smallText" margin="none">
-                Checking for updates...
-              </Paragraph>
-            </div>
-          )}
-          {!isLoading && data?.upToDate === false && (
-            <div className="flex flex-col items-center space-x-2">
-              <Info className="h-8 w-8 fill-info text-info-foreground" />
-              <Paragraph
-                className="text-info"
-                variant="smallText"
-                margin="none"
-              >
-                A new version of Fresco is available! You are{' '}
-                {data?.aheadBy ?? 0}{' '}
-                {data?.aheadBy === 1 ? 'commit' : 'commits'} behind.
-              </Paragraph>
-            </div>
-          )}
-          {!isLoading && data?.upToDate === true && (
-            <div className="flex flex-col items-center space-x-2">
-              <CheckCircle2 className="h-8 w-8 fill-success text-success-foreground" />
-              <Paragraph variant="smallText" margin="none">
-                You are up to date!
-              </Paragraph>
-            </div>
-          )}
-          {error && (
-            <div className="flex flex-col items-center space-x-2">
-              <XCircle className="h-8 w-8 fill-destructive text-destructive-foreground" />
-              <Paragraph
-                className="text-destructive"
-                variant="smallText"
-                margin="none"
-              >
-                There was an error checking for updates.
-              </Paragraph>
-            </div>
-          )}
-        </div>
-      }
+      controlArea={!data?.semVerUpdateType ? statusContent : undefined}
     >
       <Paragraph>
         You are currently running Fresco v.{env.APP_VERSION} ({env.COMMIT_HASH}
         ).
       </Paragraph>
       <Paragraph>Your unique installation ID is: {installationID}</Paragraph>
+
+      {data?.semVerUpdateType && (
+        <>
+          <Alert
+            variant={data.semVerUpdateType === 'major' ? 'default' : 'info'}
+            className="mt-4"
+          >
+            <AlertTitle>
+              {' '}
+              <div className="mt-4 flex flex-row items-center space-x-2">
+                <Info className="h-8 w-8 text-info" />
+                <Paragraph
+                  className="text-info"
+                  variant="smallText"
+                  margin="none"
+                >
+                  A new version of Fresco is available: {data.newVersion}
+                </Paragraph>
```

Comment:
This definitely shouldn't be a <Paragraph>! (1) It is already inside an `Alert.Title` (which renders a heading) and (2) you've styled a paragraph like a heading!